docs/linting/README.md

```diff
@@ -68,14 +74,30 @@ dist
 
 ## Running ESLint
 
-With that configured, open a terminal to the root of your project, and run the following command
+With that configured, open a terminal to the root of your project, and run the following command:
 
 ```bash npm2yarn
-npm run eslint . --ext .js,.jsx,.ts,.tsx
+npx eslint . --ext .js,.jsx,.ts,.tsx
 ```
```

Comment:
AFAIK npm2yarn doesn't support converting npx.

Reply:
No, because there's no `npx` equivalent in yarn. The Docusaurus team also struggled a bit on this, but we eventually settled on `npx` because it at least works for everyone. Directly below, I'm suggesting to use `npm run lint` instead, which will work for everyone.

Reply:
I'm also fine with hand-writing language tabs here, if you want to get a yarn example in.

Reply:
In this one instance I think we should use custom tabs. I don't think it's great if we use `npm2yarn` everywhere to show both, I don't think this should be the exception.

Though ideally docasaurus would just support this out-of-the-box 😉

Reply:
Yeah, we're aware of this, @armano2 has a fork in place to fix all related issues, but it got stalled for some Facebook copyrighting issues 😅 (https://github.com/facebook/docusaurus/pull/6005)